### PR TITLE
Modify agent overview page

### PR DESCRIPTION
**Pull request by matiasmoreno876**
# Description
Closes https://github.com/wazuh/wazuh-kibana-app/issues/4256

This PR will improve the Agent Overview view, fixing different issues found. All the changes were also tested in dark mode.

## Main Issue 
On some resolutions, the Agent's overview page rendered like this:
image
That render hasn´t been scrolled, but look like if it were scrolled to the bottom of the page.

### Removed the anchor to the agent details menu:
This change was done to fix the aforementioned issue. The Agent's detail is now another section on the main screen that will move on scroll, preventing padding problems when its size changes. 

**Before the changes:**

https://user-images.githubusercontent.com/42900763/191339461-f4a869da-5bd6-4478-927e-d08925429e0d.mov

**After the changes:**

https://user-images.githubusercontent.com/42900763/191335917-61ce6981-ee6f-41db-a363-486ed9aecb4a.mov

### Fixed menu buttons for small screens
This change was done to improve the experience when the App is used on a small screen, for example, from a mobile phone on vertical mode. Previously the menu wasn't displayed, and the agent name was unnecessarily trimmed.

**Before the changes:**

https://user-images.githubusercontent.com/42900763/191338866-1d2961af-0089-4a43-82a2-9dd3050ffcb6.mov

**After the changes:**

https://user-images.githubusercontent.com/42900763/191336665-3e5fb84f-1602-480c-95ce-e96944b3ea55.mov

# Tests
Scenario: Open the summary page for an agent
Given a screen width higher than 1360px
Then the elements should be shown without overlapping or issues

Scenario: Open the summary page for an agent
Given a screen width lower than 1360px and higher than 768px
Then the elements should be shown without overlapping or issues

Scenario: Open the summary page for an agent
Given a screen width lower than 768px
Then the elements should be shown without overlapping or issues

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,14 @@ All notable changes to the Wazuh app project will be documented in this file.
...
 
 - Changed the HTTP verb from `GET` to `POST` in the requests to login to the Wazuh API [#4103](https://github.com/wazuh/wazuh-kibana-app/pull/4103)
+- Improve alerts summary performance [#4376](https://github.com/wazuh/wazuh-kibana-app/pull/4376)
+- Endpoint `/agents/summary/status` response was adapted. [#3874](https://github.com/wazuh/wazuh-kibana-app/pull/3874)
+- Makes Agents Overview loading icons independent [#4363](https://github.com/wazuh/wazuh-kibana-app/pull/4363)
+- Improve Agent Overview view [#4516](https://github.com/wazuh/wazuh-kibana-app/pull/4516)
+
+### Fixed
+
+- Improves Agents Overview performance [#4363](https://github.com/wazuh/wazuh-kibana-app/pull/4363)
```
Comment (AlexRuiz7) [inline CHANGELOG.md:22]: These changes do not correspond to the PR.

The PR base branch seems to be off, and might need a rebase.

Please, contact me and we'll review it.

**Review by AlexRuiz7 — CHANGES_REQUESTED** [review]

**Review by AlexRuiz7 — APPROVED** [review]
LGTM!

**Review by Mayons95 — APPROVED** [review]
TR: :heavy_check_mark: 
<details>

[Screencast from 21-09-22 15:26:21.webm](https://user-images.githubusercontent.com/104914131/191582904-e23e4ce4-3034-4c9c-8925-fc26825b7f8e.webm)


</details>
LGTM

**Comment by Mayons95** [comment]
I found a problem with the agent name, but reviewed with @Tostti we found that is not a particular behavior related with this PR, so I decided to put the evidence of the issue here to discuss if we can open a new issue to work on this.
<details>

Screenshot from 2022-09-21 15-30-24


</details>

**Pull request merged**